### ISSUE TITLE
Poor English in `--validate-conversion-3to4` error message

### ISSUE THREAD

**Issue by Jayman2000** [godotengine/godot#65482]
### Godot version

v4.0.alpha.custom_build [019253512]

### System information

Arch Linux (IDK how to give a version for this)

### Issue description

When you run the editor with the `--validate-conversion-3to4` command-line option, can either specify a path to a Godot 3 project or have it look for a `project.godot` file in your current working directory. If you don’t specify a valid path and your current working directory does not contain a `project.godot` file, then you’ll get an error. That error is poorly worded:

<samp>ERROR: Current directory doesn't contains any Godot 3 project</samp>

### Steps to reproduce

1. Create an empty folder.
2. `cd` to that empty folder
3. Run `godot4 --validate-conversion-3to4`

### Minimal reproduction project

N/A (happens before the Project Manager loads)

**Comment by Calinou** [comment]
I suggest rewording it to:

`ERROR: Current working directory doesn't contain a `project.godot` file for a Godot 3 project.`

**Comment by Mickeon** [comment]
It seems like most messages in the file avoid adding a beginning "The". Although there definitely are several minor inconsistencies in the error messages of the converter. E.g. sometimes there's "**ERROR**:...", sometimes it just plainly says "Failed to..."
